init.rb

```diff
@@ -28,17 +28,17 @@
   description 'Plugin to generate and use issue templates for each project to assist issue creation.'
   version '0.1.1'
   author_url 'http://twitter.com/akiko_pusu'
-  requires_redmine :version_or_higher => '2.5'
+  requires_redmine version_or_higher: '2.5'
   url 'https://bitbucket.org/akiko_pusu/redmine_issue_templates'
 
-  menu :admin_menu, :redmine_issue_templates, { :controller => 'global_issue_templates', :action => 'index' },
-       :caption => :global_issue_templates
+  menu :admin_menu, :redmine_issue_templates, { controller: 'global_issue_templates', action: 'index' },

```

Comment:
__[rubocop]__
* Line is too long. [104/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)